types/schema/data/Property.ts

```diff
@@ -69,6 +71,14 @@ export interface UKProperty {
       lastUseEndDate: Date;
     };
   };
+  units?: {
+    residential: {
+      tenure: UKTenureType;
```

Comment:
Quick naming question - do we want to have "UK" / "National" just to be implicit in all these types, with subcategories based on them?

So`TenureType` is always assumed to be national unless prefixes/suffixed by a region? e.g. `GLATenureType` or `TenureTypeGLA`

Main reason to flag this would be if something is currently national (e.g. `Materials`), but later becomes more distinct for a region it would mean changing the definition of the original type + adding a new one (e.g. `MaterialsUK` + `MaterialsScotland` -  a breaking change) vs just adding a new type (`MaterialsScotland` - a minor change).

Reply:
This is a good question. In my mind, I'm using the UK prefix for specific cases when the GLA & UK categories have _no overlap_, which feels a little different than cases where the GLA categories _extend_ national categories. 

But you're right that this could very well create unnecessary breaking changes later. I'll open a follow-up re-naming PR before we do a proper release to remove the UK prefixes and make all of this consistent (it's going to touch a lot of files)!

Reply:
Yep this seems like the best approach here 👍